morphir/src/org/finos/morphir/runtime/MorphirRuntimeError.scala

```diff
@@ -1,47 +1,107 @@
 package org.finos.morphir.runtime
```

Comment:
This had to be superseded by the errors in the original `morphir/org.finos.morphir.MorphirRuntimeError` since Native.scala and NativeSDK.scala implementations use that.